modules/test/playwright/tests/users-admin-web/organizationsItemSelector.spec.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test('LPD-24824 User only sees the organizations they have permission to view', async ({
+	apiHelpers,
+	context,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	const organization1 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: 'Organization' + getRandomInt(),
+	});
+	const organization2 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: 'Organization' + getRandomInt(),
+	});
+	const user = await apiHelpers.headlessAdminUser.postUserAccount();
+	await apiHelpers.headlessAdminUser.postUserAccountByEmailAddress(
+		organization1.id,
+		user.emailAddress
+	);
+	const organizationOwnerRole =
+		await apiHelpers.headlessAdminUser.getRoleByExternalReferenceCode(
+			'Organization Owner'
+		);
+	await apiHelpers.headlessAdminUser.postOrganizationRoleUserAccountAssociation(
+		organizationOwnerRole.id,
+		user.id,
+		organization1.id
+	);
+
+	await usersAndOrganizationsPage.goToUsers();
+	await page
+		.locator(
+			`#_com_liferay_users_admin_web_portlet_UsersAdminPortlet_users_${user.id}`
+		)
+		.locator(`[aria-label="Show Actions"]`)
+		.click();
```

Comment:
this should be a dynamic locator (like a function) inside the usersAndOrganizationsPage (I think)